deepchem/feat/molecule_featurizers/roberta_tokenizer.py

```diff
@@ -0,0 +1,39 @@
+from deepchem.feat import MolecularFeaturizer
+from transformers import RobertaTokenizerFast
```

Comment:
This should be wrapped in a try/catch block since `transformers` is an optional dependency

Reply:
done!

Reply:
when I add this the CI fails since transformers is an optional dependency - should I remove the try-catch block then?